packages/modals/src/elements/DrawerModal/DrawerModal.spec.tsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef, useState, createRef, forwardRef } from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, waitFor } from 'garden-test-utils';
+import { DrawerModal, IDrawerModalProps } from './DrawerModal';
+
+describe('DrawerModal', () => {
+  const DRAWER_MODAL_ID = 'TEST_ID';
+
+  const Example = forwardRef<HTMLDivElement, IDrawerModalProps>((props, ref) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const buttonRef = useRef<HTMLButtonElement>(null);
+
+    return (
+      <>
+        <button ref={buttonRef} onClick={() => setIsOpen(true)}>
+          Open Drawer
+        </button>
+        <DrawerModal ref={ref} isOpen={isOpen} onClose={() => setIsOpen(false)} {...props}>
+          <DrawerModal.Header>title</DrawerModal.Header>
+          <DrawerModal.Close />
+          <DrawerModal.Body>body</DrawerModal.Body>
+          <DrawerModal.Footer>
+            <DrawerModal.FooterItem>
+              <button>Click</button>
+            </DrawerModal.FooterItem>
+          </DrawerModal.Footer>
+        </DrawerModal>
+      </>
+    );
+  });
+
+  Example.displayName = 'Example';
```

Comment:
Should we disable this eslint rule for spec files?

Reply:
I copy 🍝 from [TooltipModal.spec.tsx](https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/elements/TooltipModal/TooltipModal.spec.tsx#L50). I'm not sure, maybe it could be kept since it doesn't hurt and it's straight forward to do what the linter is telling us to do?

Reply:
👍 no problem if we want to leave it. This is only really useful for non-test components to show correctly in the stack-trace or React devtools